### ISSUE TITLE
Default service values

### ISSUE THREAD

**Issue by melvinotieno** [fhsinchy/tent#20]
tent start mongo starts mongo db with access-control enabled. How do I disable this option as I do not want to pass user and password to the container instance. I am running locally for development.

OS: Fedora 34

**Comment by fhsinchy** [comment]
Sadly, there is no way right now. Username and password is a requirement for majority of the services in tent. 😔

**Comment by melvinotieno** [comment]
Okay thank you. I can work with that still. Thank you for this tool. It actually does help a lot rather than doing it with podman alone

**Comment by fhsinchy** [comment]
Thanks for the idea though. I may as well add an `--insecure` flag to start the services without any password. I'll look into it tomorrow morning.

**Comment by melvinotieno** [comment]
Yes that would be great. That way, I won't have to keep setting up a user every time I create a new database.

**Comment by melvinotieno** [comment]
reopened just because there could be a fix for it

**Comment by melvinotieno** [comment]
Any progress on this?

**Comment by fhsinchy** [comment]
No not yet. I'm a little busy these days sadly. Also need to update the codebase with the latest tagged version of their bindings. Right now, I can't give any ETA either.

**Comment by fhsinchy** [comment]
Hello again @melvinotieno I also added the `--insecure` flag in another of my [commit](https://github.com/fhsinchy/tent/commit/6948836c31d6b26146e3fe99d83136ea82329dea).
